README.md

```diff
@@ -6,7 +6,6 @@
   <a href="https://radar.com">Website</a> |
   <a href="https://radar.com/blog">Blog</a> |
   <a href="https://radar.com/documentation">Documentation</a> |
-  <a href="https://radar.com/documentation/faq">FAQ</a> |
```

Comment:
why's this removed?

Reply:
Link is broken. We could link to https://radar.com/documentation/faqs instead, but think a top-level link to /documentation is prob enough